src/test/java/racingcar/RacingGameTest.java

```diff
@@ -1,10 +1,10 @@
 package racingcar;
```

Comment:
```suggestion
package racingcar.model;
```

보통 테스트 코드의 패키지 구조는 프로덕션 코드의 패키지 구조와 동일하게 가져갑니다. 따라서 위와 같이 지정하는 것이 적절해 보여요. 다른 테스트 클래스들도 마찬가지로요~